app/pages/reservation/reservation-information/_reservation-information.scss

```diff
@@ -18,3 +18,67 @@
     margin-bottom: 10px;
   }
 }
+
+.app-InternalReservationForm__checkbox {
+  display: block;
+  position: relative;
+  padding-left: 35px;
+  padding-bottom: 12px;
+  margin-bottom: 12px;
+  cursor: pointer;
+  font-size: 22px;
```

Comment:
use sass variable from opencity.design 
(Check how font-size is used across varaamo)

Reply:
This hacky css is removed.